CONTRIBUTING.rst

```diff
@@ -223,13 +223,13 @@ Running System Tests
    $ gcloud beta emulators datastore env-init
    export DATASTORE_LOCAL_HOST=localhost:8417
    export DATASTORE_HOST=http://localhost:8417
-   export DATASTORE_DATASET=gcloud-settings-app-id
-   export DATASTORE_PROJECT_ID=gcloud-settings-app-id
+   export DATASTORE_DATASET=google-cloud-settings-app-id
+   export DATASTORE_PROJECT_ID=google-cloud-settings-app-id
```

Comment:
Those are just placeholders. For me locally the value is `bossylobster` (my default project)